tests/testsuite/rust_version.rs

```diff
@@ -124,6 +124,105 @@ fn rust_version_bad_pre_release() {
         .run();
 }
 
+#[cargo_test]
+fn rust_version_x_wildcard() {
```

Comment:
I wonder if we should move most of the parsing test case into `cargo-util-schemas` as unit tests. They may be better living closer.

Reply:
Migrated them